internal/osbuild2/ostree_config_stage.go

```diff
@@ -0,0 +1,27 @@
+package osbuild2
+
+type ConfigOptions struct {
```

Comment:
We should probably rename this to something like `OSTreeConfig` to avoid future collisions with other config structs in the package and to make it easier to know which stage it belongs to.  The name can get awkward with all these structs but it's generally better to pseudo-namespace them I think.

Reply:
fixed!